database/migrations/default/1761935469547_units_invloved/up.sql

```diff
@@ -0,0 +1,58 @@
+CREATE OR REPLACE VIEW unit_types_involved_view AS
+WITH
+    unit_types AS (
+        SELECT
+            crash_pk,
+            CASE
+                WHEN unit_desc_id = 3 THEN 'Bicycle'
+                WHEN unit_desc_id = 4 THEN 'Pedestrian'
+                WHEN veh_body_styl_id = 71
+                OR veh_body_styl_id = 90 THEN 'Motorcycle'
+                WHEN veh_body_styl_id = 177 THEN 'E-Scooter'
+                WHEN unit_desc_id = 1
+                AND veh_body_styl_id IN (0, 9, 23, 30, 47, 69, 87, 92, 100, 103, 104, 105, 106, 107, 109) THEN 'Car'
+                WHEN unit_desc_id = 1 AND veh_body_styl_id IS NULL THEN 'Car'
+                -- assign TOWED/TRAILER as car
+                WHEN unit_desc_id = 6 then 'Car'
```

Comment:
Towed/Trailer is used for example when a car crashes into a trailer being towed by a vehicle? 

I probably have mentioned it before, but I still find it odd that theres no distinction for trucks / semis / 18 wheelers. A mini cooper and a 18 wheeler are both categorized the same.